docs/index.md

```diff
@@ -9,12 +9,18 @@ description: |-
 
 The HCP provider provides resources to manage [HashiCorp Cloud Platform](https://cloud.hashicorp.com/) (HCP) resources.
 
+~> **Known Issue** There is an issue with v0.4.0 of the HCP Provider in which existing Consul clusters that do not specify size will be force-recreated on the next Apply. Please upgrade to the patch v0.4.1 to avoid this issue.
+
```

Comment:
I think I understand the intent of this message, but from the user's perspective it may be a little weird. What we will end up with is a `v0.4.2` when this gets released, which will mention that the user should use `v0.4.1`, because `v0.4.0` has an issue. But then when the user selects `v0.4.0` or `v0.4.1` from the version selector, they will not see any messages related to known issues. I'm wondering if we will instead need to rely on a more generic message if we really need something like this, eg. `Please refer our [CHANGELOG]() for critical fixes`

Reply:
I like the idea of adding the generic message, I can do that. @xargs-P wants to keep the specific 0.4.0 warning in for 30 days, then we'll remove it. We're adding the same 30-day warning to the CLI (which will show up when the provider initializes). We want to be extra noisy about that issue, since it could result in data loss.